src/Repository/ForumRepository.php

```diff
@@ -86,11 +89,17 @@ public function findForumsByPage(int $page, string $sortBy) {
      */
     public function findSubscribedForumNames(User $user) {
         /* @noinspection SqlDialectInspection */
-        $dql =
-            'SELECT f.id, f.name FROM '.Forum::class.' f WHERE f IN ('.
-                'SELECT IDENTITY(fs.forum) FROM '.ForumSubscription::class.' fs WHERE fs.user = ?1'.
-            ') ORDER BY f.normalizedName ASC';
-
+        if ($user->isAdmin()) {
+            $dql =
+                'SELECT f.id, f.name FROM '.Forum::class.' f WHERE f IN ('.
+                    'SELECT IDENTITY(fs.forum) FROM '.ForumSubscription::class.' fs WHERE fs.user = ?1'.
+                ') ORDER BY f.normalizedName ASC';
+        } else {
+            $dql =
+                'SELECT f.id, f.name FROM '.Forum::class.' f WHERE f IN ('.
+                    'SELECT IDENTITY(fs.forum) FROM '.ForumSubscription::class.' fs WHERE fs.user = ?1 and f.id > 0'.
+                ') ORDER BY f.normalizedName ASC';
+        }
```

Comment:
let's refactor this into 

```
$nonAdminClauseIfAny = 'AND f.id > 0';
if ($user->isAdmin()) {
  $nonAdminClauseIfAny = '';
}

$dbql = BLABLA . $nonAdminClauseIfAny
```

this will prevent future edited-in-one-place-forgot-another bug and make it easier to distinguish admin/non-admin differences.

Actually you can probably extract #nonAdminClauseIfANy
to a function and use it everywhere sintead of 'f.id >0 ' string
Should minimize code-dupe

Reply:
Meant to rewrite this before PR sorry